CONTRIBUTING.md

```diff
@@ -72,71 +92,79 @@ By making a contribution to this project, I certify that:
 
 ### Git Commit Messages
 
+Commit message should follow the template `:<emoji>: <subject>` with `<emoji>` being the name of the relevant [emoji](#emoji-cheatsheet) describing the changes (without wrapping `::`) and `subject` being the description of changes. Commit messages may have multiple emoji. Commit messages should follow the following guidelines:
```

Comment:
This is totally fine....but the actual emoji-does-xyz needs to be fleshed out to match what's happening in our semvar module. A couple times I've had to dig in to the code on semvar to figure out which emoji would trigger what. We should have the specific emoji that do XYZ in the readme of semvar (or somewhere more apparent...or maybe in this doc?).